### DISCUSSION TITLE
Voyager not installing in Laravel 11 Project

### DISCUSSION THREAD

**Discussion by achowdhury39** [thedevdojo/voyager#5856] [General]
Voyager not installing in Laravel 11 Project.
Have there any way to install in Laravel 11 project?

**Comment by soinalastudio** [comment]
The same too here, so I installed it in Laravel 10...

**Comment by Emerica** [comment]
https://github.com/thedevdojo/voyager/issues/5846

**Reply by Emerica** [reply]
It's becoming my opinion that the project needs trusted maintainer(s) first.
The founders have become very quiet and development has stagnated, now even if PR's are created there is limited ability to safely merge them without forking out or trying to eat some of @tnylea 's time getting things implemented. I'm assuming that's limited from what I've seen so far.  Safely is also a key word there, XZ should be a wake up call for all, and we don't need another PHP-Nuke. 

We should at the very least outline all the issues that present a challenge for Laravel 11 and discuss the options to solve them.
The above issue mentions DBAL being an issue, but there are no details and I'm not familiar with how Voyager makes use of the library.
That or new issue(s) might be a good place to continue those technical discussions, PR's can be made and put in Tony's view.
Short of that we're forking and fracturing and I'd rather see one project continue.
